content/en/platform/corda/4.11/enterprise/node/collaborative-recovery/ledger-recovery/recovery-using-shell.md

```diff
@@ -0,0 +1,58 @@
+---
+date: '2023-11-08'
+menu:
+  corda-enterprise-4-11:
+    parent: corda-enterprise-4-11-ledger-recovery
+tags:
+- ledger recovery
+title: Ledger recovery using the shell
+weight: 800
+---
+
+# Ledger recovery using the shell
+
+The following examples show the different ways to use the Ledger Recovery flow from the Corda node shell.
+
+* Run Ledger Recovery for a given time window and using all available nodes in the network:
+
+  ```bash
+  flow start EnterpriseLedgerRecoveryFlow timeWindow: { fromTime: "2023-07-19T09:00:00Z", untilTime: "2023-07-19T09:00:00Z" }, useAllNetworkNodes: true
+  ```
+
+* Run Ledger Recovery without actually performing reconciliation (for example, `dryRun = true`) with detailed
+  record/transaction-level output (`verboseLogging = true`) for a given time window and using all available nodes in the network:
+
+  ```bash
+  flow start EnterpriseLedgerRecoveryFlow timeWindow: { fromTime: "2023-07-19T09:00:00Z", untilTime: "2023-07-19T09:00:00Z" }, dryRun: true, verboseLogging: true, useAllNetworkNodes: true
+  ```
+
+* Run Ledger Recovery for a given time window and specifying a single specific peer recovery node:
+
+  ```bash
+  flow start EnterpriseLedgerRecoveryFlow timeWindow: { fromTime: "2023-10-12T19:00:00Z", untilTime: "2023-12-12T22:00:00Z" }, recoveryPeer:  "O=Bank B, L=New York, C=US"
+  ```
+
+* Run Ledger Recovery for a given time window and specifying several peer recovery nodes:
+
+  ```bash
+  flow start EnterpriseLedgerRecoveryFlow timeWindow: { fromTime: "2023-10-12T19:00:00Z", untilTime: "2023-12-12T22:00:00Z" }, recoveryPeers: ["O=Bank B, L=New York, C=US", "O=Bank C, L=Chicago, C=US"]
+  ```
+
+* Run Ledger Recovery for a given time window and single peer recovery node, and perform finality flow recovery of any encountered `IN_FLIGHT` transactions:
+
+  ```bash
+  flow start EnterpriseLedgerRecoveryFlow timeWindow: { fromTime: "2023-10-12T19:00:00Z", untilTime: "2023-12-12T22:00:00Z" }, recoveryPeer:  "O=Bank B, L=New York, C=US", alsoFinalize: true
+  ```
+
+## Corda node shell sample output
```

Comment:
I think you need another H2 for the examples (or remove this one).

Reply:
The H1 introduces the examples so not sure if we need another H2. The H2 for the sample output makes it stand out so I wouldn't remove it.

Reply:
It's probably missing from our style guide but usually, it's not good practice to only have one section at any level... https://academia.stackexchange.com/questions/12190/is-it-bad-style-if-a-section-has-exactly-one-subsection